<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QWidget" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(115, 210, 22);</string>
  </property>
  <widget class="QLabel" name="lbl_max_position">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_playPause">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border:2px solid #424242;}</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_slash">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>10</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_shuffle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	color: rgb(115, 210, 22);
	background: none;
	font: 24px;
}

QCheckBox::indicator {
	width: 24px;
	height: 24px;
}</string>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_play_on_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>200</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	color: rgb(115, 210, 22);
	background: none;
	font: 24px;
}

QCheckBox::indicator {
	width: 24px;
	height: 24px;
}</string>
   </property>
   <property name="text">
    <string>Play on start</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_current_position">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sldr_volume_max">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>50</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_previous">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border:2px solid #424242;}</string>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_artist">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 16pt;
}</string>
   </property>
   <property name="text">
    <string>Motionless in White</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {border:2px solid #424242;}</string>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_mute">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	color: rgb(115, 210, 22);
	background: none;
	font: 24px;
}

QCheckBox::indicator {
	width: 24px;
	height: 24px;
}</string>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gv_cover">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGraphicsView {border:2px solid #424242;}</string>
   </property>
  </widget>
  <widget class="QSlider" name="sldr_volume">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>21</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::add-page {
background: #22b52a;
border: 2px solid #424242;
height: 10px;
border-radius: 4px;
}

QSlider::handle {
background:#22b52a;
border: 2px solid #424242;
width: 12px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sldr_volume_min">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>50</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_song">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 16pt;
}</string>
   </property>
   <property name="text">
    <string>Disguise</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sldr_volume_current">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>50</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_album">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 16pt;
}</string>
   </property>
   <property name="text">
    <string>Disguise</string>
   </property>
  </widget>
  <widget class="QSlider" name="sldr_song">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::add-page {
background: #22b52a;
border: 2px solid #424242;
height: 10px;
border-radius: 4px;
}

QSlider::handle {
background:#22b52a;
border: 2px solid #424242;
width: 12px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
